app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala

```diff
@@ -125,17 +125,8 @@ class TaxAccountConnector @Inject()(httpHandler: HttpHandler, servicesConfig: Se
         TaiTaxAccountFailureResponse(e.getMessage)
     }
 
-  def totalTax(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TaiResponse] =
+  def totalTax(nino: Nino, year: TaxYear)(implicit hc: HeaderCarrier): Future[TotalTax] =
     httpHandler.getFromApiV2(totalTaxUrl(nino.nino, year)) map (
-      json => TaiSuccessResponseWithPayload((json \ "data").as[TotalTax])
-    ) recover {
-      case e: NotFoundException =>
-        logger.warn(s"Total tax - No tax account information found: ${e.getMessage}")
-        TaiNotFoundResponse(e.getMessage)
-      case e: UnauthorizedException => TaiUnauthorisedResponse(e.getMessage)
-      case e: Exception =>
-        logger.warn(s"Couldn't retrieve total tax for $nino with exception:${e.getMessage}")
```

Comment:
I see you have added in the log for notFound at the controller level but not this one, should it be added back in?